### ISSUE TITLE
Weird labels in gluster dashboard

### ISSUE THREAD

**Issue by fbalak** [Tendrl/dashboard#375]
I have one imported gluster cluster with no volumes. In gluster dashboard I see some weird labels (config.title, currentValue, ...) in place of valid data as seen on picture. 
![dashboard_weird_labels](https://cloud.githubusercontent.com/assets/22369801/26453089/d078da36-4162-11e7-9932-af9b8dd17e7b.png)
```
http://hostname.com/api/1.0/monitoring/system/gluster
{"stats":{"hosts_count":{"down":0,"crit_alert_count":0,"total":4,"warn_alert_count":0},"sds_det":{"connection_active":0,"volume_status_wise_counts":{"down":0,"warning_alerts":0,"degraded":0,"total":0,"critical_alerts":0},"most_used_volumes":[],"most_used_bricks":[],"throughput":19397.75,"services_count":{"glusterd":{"not_running":0,"running":4},"tendrl-gluster-integration":{"not_running":0,"running":4},"tendrl-node-agent":{"not_running":0,"running":4}},"connection_count":0,"brick_status_wise_counts":{"warning_alerts":0,"critical_alerts":0,"total":0,"stopped":0}},"cluster_count":{"status":{"healthy":1,"total":1},"warning_alerts":0,"critical_alerts":0,"near_full":0},"utilization":{"total":0,"percent_used":0,"used":0},"sds_type":"gluster"}}
```
```
http://hostname/api/1.0/monitoring/system/gluster/utilization
{"stats":[]}
```
There should be a proper values or error message in case of missing or broken data.

Tested with:
tendrl-performance-monitoring-1.3.0-1.el7.centos.noarch
tendrl-api-1.3.0-1.el7.centos.noarch
tendrl-node-agent-1.3.0-1.el7.centos.noarch
tendrl-commons-1.3.0-1.el7.centos.noarch
tendrl-dashboard-1.3.0-1.el7.centos.noarch

**Comment by gnehapk** [comment]
@fbalak Its fixed and available at tendrl-dashboard-1.3.0-1.el7.centos.noarch.rpm. Please verify it.

**Comment by fbalak** [comment]
It seems ok.
Tested with:
tendrl-dashboard-1.3.0-1.el7.centos.noarch
tendrl-performance-monitoring-1.3.0-1.el7.centos.noarch
tendrl-api-1.3.0-1.el7.centos.noarch
